src/apps/AppCenter/AppCard.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {
+  Card,
+  GU,
+  IconExternal,
+  Tag,
+  textStyle,
+  unselectable,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import AppIcon from '../../components/AppIcon/AppIcon'
+
+const AppCard = React.memo(function AppCard({
+  onClick,
+  link,
+  icon,
+  name,
+  tag,
+  description,
+}) {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small'
+  const handleClick = useCallback(() => onClick(link), [onClick, link])
+
+  return (
+    <Card onClick={onClick} css={onClick ? 'display: block;' : ''}>
+      <CardMain compactMode={compactMode}>
+        {link && (
+          <StyledIconExternal
+            compactMode={compactMode}
+            theme={theme}
+            link={link}
+          />
+        )}
+        <Icon compactMode={compactMode}>{icon}</Icon>
+        <Name compactMode={compactMode}>{name}</Name>
+        <TagWrapper compactMode={compactMode} link={link}>
+          <Tag mode="indicator">{tag}</Tag>
+        </TagWrapper>
+        <Description theme={theme} compactMode={compactMode}>
+          {description}
+        </Description>
+      </CardMain>
+    </Card>
+  )
+})
+
+AppCard.propTypes = {
+  onClick: PropTypes.func,
+  link: PropTypes.string,
+  icon: PropTypes.node.isRequired,
+  name: PropTypes.string.isRequired,
+  tag: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+}
+
+const CardMain = styled.section`
+  ${unselectable};
+  position: relative;
+  overflow: hidden;
+  height: 100%;
+  width: 100%;
+  white-space: initial;
+
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+          display: grid;
+          grid-template-columns: auto 1fr auto;
+          grid-template-rows: auto auto auto;
+          grid-template-areas:
+            "empty topright"
+            "icon title"
+            "icon description";
+          padding: ${1.5 * GU}px ${1.5 * GU}px ${4 * GU}px ${3 * GU}px;
+        `
+      : `
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          padding-top: ${5 * GU}px;
+        `}
+`
+
+const Icon = styled.div`
+  height: ${9 * GU}px;
+  width: ${9 * GU}px;
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+        grid-area: icon;
+        margin-right: ${1.5 * GU}px;
+        width: 100%;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: flex-start;
+      `
+      : `
+        margin-bottom: ${2 * GU}px;
+      `}
+`
+
+const Name = styled.p`
+  display: flex;
+  width: 100%;
+  ${textStyle('title4')}
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+        grid-area: title;
+        align-self: flex-end;
+      `
+      : `
+        justify-content: center;
+        margin-bottom: ${1 * GU}px;
```

Comment:
Super subjective, but perhaps one whole GU is a too much vertical spacing 😄; what do you think about just 0.25 or 0.5?

Reply:
Yes! I was hesitant of using less than one `GU` but I'm glad you mentioned it.

Reply:
Fixed here: 6b117a06

Reply:
I don't know the specifics on how much we should adhere to GU, but there's lots of places where we use fractional GU for small adjustments so it's probably fine 🤷‍♂. At the very least 0.5 GU is quite common.

Reply:
Sure, I wanted to avoid a back and forth with this little detail but `0.5` does seem fine and I think we can use it for now and leave more fine tunning for a next iteration (we don't really have designs for this).